src/token/index.js

```diff
@@ -0,0 +1,224 @@
+import { txType } from '../tx/'
+import * as crypto from '../crypto/'
+import { api } from '../client/'
+const MAXTOTALSUPPLY = 9000000000000000000
+
+const validateSymbol = (symbol)=>{
+  if(!symbol) {
+    throw new Error("suffixed token symbol cannot be empty")
+  }
+
+  const splitSymbols = symbol.split('-')
+
+  //check length with .B suffix
+  if(!/^[a-zA-z\d]{3,10}$/.test(splitSymbols[0])) {
+    throw new Error("symbol length is limited to 3~10")
+  }
+}
+
+const validateAmount = async (amount, symbol, fromAddress, httpClient, type="free") => {
+  if(amount <= 0 || amount > MAXTOTALSUPPLY){
+    throw new Error("invalid amount")
+  }
+
+  try {
+    const { result } = await httpClient.request("get", `${api.getAccount}/${fromAddress}`)
```

Comment:
This will slow down the calls that are using this, but if it is just `freeze`, `unfreeze`, `burn`, `issue` it is probably ok.

Reply:
only freeze, unfreeze, burn